src/main.ts

```diff
@@ -109,31 +109,33 @@ export async function runAction(): Promise<any> {
 
   try {
     const executionGraphId = await createPipeline(config)
-    core.info(`Created pipeline with id ${executionGraphId}.`)
+    core.info(
+      `Created pipeline with id ${executionGraphId}. Downloading pipeline details from ${getDownloadVibPublicUrl()}/v1/execution-graphs/${executionGraphId}`
```

Comment:
Are we really downloading something? I don't think so. I would use something like _Go to the pipeline detail_

Reply:
Done